### PR TITLE
Adding some information about running the project in Docker

### PR DESCRIPTION
**Pull request by saevarom**
This took me a couple of hours to figure out. 
Might save someone else some time.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,10 +4,11 @@ node_modules/
...
+.env
 
 apps/frontend/static/
 apps/guide/settings/local.py
 
 /media/
 /static/
-.coverage
\ No newline at end of file
+.coverage
```
Comment (lb-) [inline .gitignore:14]: Cna you re-add the end of line?

File: README.md [hunk 2]
```diff
@@ -61,6 +61,20 @@ To activate Poetry's virtual environment, run:
...
+    ALLOWED_HOSTS=localhost
+    PORT=8000
+    SECRET_KEY=some-random-secret
+    DJANGO_SETTINGS_MODULE=apps.guide.settings.dev
+
+2. Build the image by running the `make docker-build` command.
+3. Run the container with `make docker-run`.
+4. Run the init script in the container: `make docker-init`
+5. You should now have access to the project in your browser at `http://localhost:8000`
+
```
Comment (lb-) [inline README.md:77]: This will need to have prettier run on it. `make format`

File: README.md [hunk 3]
```diff
@@ -61,6 +61,20 @@ To activate Poetry's virtual environment, run:
 
     poetry shell
 
+### Setting up Wagtail guide to develop on Docker
```
Comment (lb-) [inline README.md:64]:
```suggestion
### Setting up development with Docker
```

Nitpicky - just removing the name of this project in the title - not really needed.

File: .gitignore (hunk 1 above)
Comment (saevarom) [reply .gitignore:14]: This one I edited in vim, which removes new-lines apparently

File: Makefile [hunk 4]
```diff
@@ -30,3 +32,15 @@ backend:
...
+	docker build -t guide:latest --build-arg POETRY_INSTALL_ARGS="" -f Dockerfile .
+
+docker-run:
+	docker run --name guide_latest -p ${PORT}:${PORTs} --env-file .env guide:latest sh -c 'poetry run python manage.py runserver 0.0.0.0:${PORT}'
+
+docker-exec:
+	docker exec -it guide_latest /bin/bash
+
+docker-init:
+	docker exec -it guide_latest /bin/bash -c "make backend"
```
[the remark this answers is not in the document]
Comment (saevarom) [reply Makefile:49]: Here it was just me not adding the new-line

File: README.md (hunk 2 above)
Comment (saevarom) [reply README.md:77]: I'm having some trouble with my yarn installation, but I ran it through preview mode on Github and made some changes

Comment (lb-) [reply README.md:77]: The lint failure could be other code sorry. 

https://github.com/wagtail/guide/pull/173

File: Makefile (hunk 4 above)
[the remark this answers is not in the document]
Comment (lb-) [reply Makefile]: Thanks.

**Comment by thibaudcolas** [comment]
Thank you @saevarom, looks solid to me.

**Pull request merged**